ci/jenkins/jobs/projects.yaml

```diff
@@ -237,6 +255,88 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: whole-conformance-pending-label
+          node: null
+          description: 'This is for marking PR as pending for whole conformance test.'
+          builders:
+            - builder-pending-label
+          trigger_phrase: null
+          allow_whitelist_orgs_as_admins: false
+          admin_list: []
+          org_list: []
+          white_list: []
+          only_trigger_phrase: false
+          trigger_permit_all: true
+          status_context: jenkins-whole-conformance
+          status_url: --none--
```

Comment:
what's the difference between `--none--` and `null`?

Reply:
`null` applies to e2e, conformance.. so Jenkins will automatically put a link to the build.

`--none--` means there's no link.